SCORER.md

```diff
@@ -21,11 +25,175 @@ You may need to build your own scorer - your own _language model_ if:
 * You are training a speech recognition model for a particular domain - such as technical words, medical transcription, agricultural terms and so on
 * If you want to improve the accuracy of transcription
 
-DeepSpeech supports the _optional_ use of an external scorer - if you're not sure if you need to build your own scorer, stick with the built-in one to begin with.
+**DeepSpeech supports the _optional_ use of an external scorer - if you're not sure if you need to build your own scorer, stick with the built-in one to begin with**.
 
 ### Building your own scorer
 
-Building your own scorer is beyond the scope of the DeepSpeech Playbook, but the [DeepSpeech documentation covers how to do this](https://deepspeech.readthedocs.io/en/latest/Scorer.html). There are built-in scripts with DeepSpeech which make building an external scorer - a _language model_ - easier.
+The process for building your own scorer comprises the following steps:
+
+1. Having, or preparing, a text file (in `.txt` or `.txt.gz` format), with one phrase or word on each line. If you are training a speech recognition model for a particular domain - such as technical words, medical transcription, agricultural terms etc, then they would appear in the text file.
+
+2. Using the `generate_lm.py` script which is distributed with DeepSpeech, along with the text file, to create two files, called `lm.binary` and `vocab-500000.txt`.
```

Comment:
Technically  @nmstoker is working on a PR to even allow merging multiple files at once through this script

Reply:
might be worth emphasizing that to run `generate_lm.py`, following training setup is recommended

Reply:
> Technically @nmstoker is working on a PR to even allow merging multiple files at once through this script

Great! I can easily update the PlayBook once the PR is merged. 

> might be worth emphasizing that to run generate_lm.py, following training setup is recommended

Good call, I have added this.